<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="LoginCenterWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="UserNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>275</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>User Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="UserNameText">
        <property name="maximumSize">
         <size>
          <width>275</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="NWradioButton">
      <property name="text">
       <string>Network</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="IORadioButton">
      <property name="text">
       <string>Input / Output</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MonitorButton">
      <property name="text">
       <string>Monitor</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
